docs/book/src/managed/managedcluster.md

```diff
@@ -96,7 +96,7 @@ clusterctl generate cluster ${CLUSTER_NAME} --kubernetes-version ${KUBERNETES_VE
 kubectl apply -f cluster.yaml
 
 # check status of created resources
-kubectl get cluster-api -o wide
+kubectl get cluster my-cluster -o wide
 ```
```

Comment:
Actually, I think this command is intended to show all resources tagged with "cluster-api," not just the cluster resource. This works for me (I use it a lot).

Reply:
TIL, this is quite handy. Out of interest, how does this work? Because I couldn't find any labels for `cluster-api` in the objects returned :thinking:

Reply:
I *think* there's an `app=cluster-api` label on each of the resources in the management cluster, and the api server (or maybe `kubectl` itself) have this fallback behavior. I'm not sure where it's documented actually.

I was quite surprised when someone typed this command in an online CAPI meeting and it just did the right thing. 😄

Reply:
I think it's this `category=cluster-api` is the magic to enable this: https://github.com/kubernetes-sigs/cluster-api/blob/65fbabe1cf55dc6a2f13b2afde0d3024ee04d35c/api/v1beta1/cluster_types.go#L508

That seems to make this change to the CRDs: https://github.com/kubernetes-sigs/cluster-api/blob/65fbabe1cf55dc6a2f13b2afde0d3024ee04d35c/config/crd/bases/cluster.x-k8s.io_clusters.yaml#L11-L12

Reply:
Indeed https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#categories Thanks for the context :+1: